src/services/api/DatagouvfrAPI.js

```diff
@@ -48,6 +48,17 @@ export default class DatagouvfrAPI {
     return await this.makeRequestAndHandleResponse(url)
   }
 
+  /**
+   * Get an entity's detail from its id, without wrapper
+   *
+   * @param {string} entity_id
+   * @returns {import("../../composables/fetch").ComposableFetchResult}
+   */
+  async _get (entity_id) {
```

Comment:
Hum, not sure, it might be my Python instincts kicking in :p But I think I remember some Java stuff along those lines, for private methods?

Reply:
Java has `private`, so I suspect it's your python instinct :p 
I was more referring to the "no wrapper" aspect of the function actually, but it's moot since it wasn't the semantics you meant.